### PR TITLE
explicitly specify 7z

### PR DESCRIPTION
**Pull request by musm**

### CODE REVIEW

**Comment by musm** [comment]
https://github.com/JuliaLang/julia/pull/21540

File: src/tzdata/archive.jl [hunk 1]
```diff
@@ -1,5 +1,9 @@
 import Compat: @static, is_windows
 
+if is_windows()
+    const exe7z = joinpath(JULIA_HOME, "7z.exe")
```
Comment (omus) [inline src/tzdata/archive.jl]: Can you rename this to `7Z`?

Comment (musm) [inline src/tzdata/archive.jl]: unfortunately not, can't start with a number `7Z = 7*Z`

Comment (omus) [inline src/tzdata/archive.jl]: Ah, that slipped my mind.

**Pull request merged**